tap_easyecom/client.py

```diff
@@ -19,7 +19,6 @@ def get_next_page_token(
     ):
         """Return a token for identifying next page or None if no more pages."""
         res_json = response.json()
-        next_url = res_json.get("nextUrl", res_json.get("data", {}).get("nextUrl"))
```

Comment:
lol you're referencing `next_url` on the next line. did you even run this to test?